server/cmd/server/modules.go

```diff
@@ -0,0 +1,5 @@
+package main
+
+/*import (
+	"github.com/google/wire"
+)*/
```

Comment:
delete or uncomment it

Reply:
I cannot uncomment it, because we are not using wire yet, but I think we should leave it to show what this file will contain in future comments.